packages/management/addons-manager/README.md

```diff
@@ -0,0 +1,43 @@
+# addons-manager Package
+
+Addon-manager generates App CRs for addons based on TKR/BOM, KubeadmControlPlane and CLI inputs provided during cluster creation.
+It then creates these App CRs on management and workload clusters which then get reconciled by a local kapp-controller.
+
+## Components
+
+Client Library
+Addon Controller
+
+## Usage Example
+
+The following is a basic guide for getting started with addons-manager.
+
+Management cluster
+
+1. “tkg init” to create a management cluster.
```

Comment:
suggestion: I guess we no longer use `tkg init` for mgmt cluster creation. Is it supposed to be `tanzu management-cluster create` ?

Reply:
updated to tanzu cli cmd